Lib/test/decimaltestdata/abs.decTest

```diff
@@ -20,7 +20,7 @@
 version: 2.59
 
 -- This set of tests primarily tests the existence of the operator.
--- Additon, subtraction, rounding, and more overflows are tested
```

Comment:
This is an external source.  I would prefer not to maintain our own version, even for typos.

Reply:
reverted.